deploy/k8s/overlays/prod/kustomization.yml

```diff
@@ -0,0 +1,30 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: labelstudio
+
+namePrefix: labelstudio-
+
+bases:
+  - ../../base/
+
+resources:
+  - ingress.yml
+
+patchesStrategicMerge:
+  - env.yml
+
+commonLabels:
+  stage: production
+
+secretGenerator:
+- name: labelstudio
+  env: labelstudio.env
+
+images:
+  - name: postgres
+    newTag: '11.5'
+  - name: nginx
+    newTag: 'latest'
+  - name: heartexlabs/label-studio
+    newTag: '1.0.2'
```

Comment:
Please switch to the latest available labelstudio release.

Reply:
will do also, but can you confirm to me there are no other changes that should follow? I don't see any critical change since 1.0.2 neither in the deploy directory nor in the docker-compose files but just to be sure. Thanks again!